### PR TITLE
[CDAP-9058] Display loading icon behind column dropdown in DataPrep

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-9058

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/DataPrepLoading/DataPrepLoading.scss [hunk 1]
```diff
@@ -24,6 +24,7 @@
     background-color: rgba(255, 255, 255, 0.7);
     color: #666666;
     font-size: 72px;
+    z-index: 10000;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrep/DataPrepLoading/DataPrepLoading.scss]: This loading z-index should be less than system down modal

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by tonybach** [comment]
What this looks like after the changes:

screen shot 2017-03-29 at 6 18 58 pm

screen shot 2017-03-29 at 6 16 59 pm

**Comment by tonybach** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC6191-1

Showed behavior to @bdmogal in-person and got an LGTM. Merging now.

**Pull request merged**